.babelrc

```diff
@@ -1,9 +1,8 @@
 {
 	"presets": [
-		[ "latest", {
-			"es2015": {
-				"modules": false
```

Comment:
Hm, interesting.  At one point I was getting an error until I removed this (http://stackoverflow.com/a/42834190) but now it is working fine.